.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -31,6 +31,7 @@ assignees: ''
 * **ESLint Version:**
 * **Node Version:**
 * **npm Version:**
+* **Operating System:** 
```

Comment:
currently, running `eslint --env-info` does not print the OS; but it seems good to have.

Reply:
I’m assuming people know what operating system they are using. :)